cranelift/codegen/src/isa/zkasm/abi.rs

```diff
@@ -294,8 +294,15 @@ impl ABIMachineSpec for ZkAsmMachineDeps {
         if amount == 0 {
             return insts;
         }
-        insts.push(Inst::AdjustSp {
-            amount: amount as i64,
+        // FIXME: is this right/sufficient for stack growing up
```

Comment:
I think it is. And see ``reserve``\``release`` is much better than think every time about what positive\negative value in ``adjust`` mean